packages/connect-query/src/use-infinite-query.ts

```diff
@@ -14,24 +14,66 @@
 
 import type { Message, PartialMessage } from "@bufbuild/protobuf";
 import type { ConnectError, Transport } from "@connectrpc/connect";
+import {
+  type ConnectInfiniteQueryKey,
+  type ConnectInfiniteQueryOptions,
+  createInfiniteQueryOptions,
+  type DisableQuery,
+  type MethodUnaryDescriptor,
+} from "@connectrpc/connect-query-core";
 import type {
   InfiniteData,
+  UseInfiniteQueryOptions as TSUseInfiniteQueryOptions,
   UseInfiniteQueryResult,
+  UseSuspenseInfiniteQueryOptions as TSUseSuspenseInfiniteQueryOptions,
   UseSuspenseInfiniteQueryResult,
 } from "@tanstack/react-query";
 import {
   useInfiniteQuery as tsUseInfiniteQuery,
   useSuspenseInfiniteQuery as tsUseSuspenseInfiniteQuery,
 } from "@tanstack/react-query";
 
-import type {
-  CreateInfiniteQueryOptions,
-  CreateSuspenseInfiniteQueryOptions,
-} from "./create-use-infinite-query-options.js";
-import { createUseInfiniteQueryOptions } from "./create-use-infinite-query-options.js";
-import type { MethodUnaryDescriptor } from "./method-unary-descriptor.js";
 import { useTransport } from "./use-transport.js";
-import type { DisableQuery } from "./utils.js";
+
+/**
+ * Options for useInfiniteQuery
+ */
+export type UseInfiniteQueryOptions<
```

Comment:
This just moved from `create-use-infinite-query-options` and just extends to `ConnectInfiniteQueryOptions` like from before

Reply:
V2 changed all this, no longer applicable